Jenkinsfile

```diff
@@ -1,138 +0,0 @@
-pipeline {
```

Comment:
Is there a reason to remove this? We can continue running that, just not make the result block any PR.

Reply:
If you expect something to fail, I don't think there's any point in running it at all - it's just noise you have to sift through.

I'm also not done with this PR yet, at the moment all I'm doing is trying to figure out how to make the docker build faster!

Reply:
I've re-added the file. I think we can disable the webhooks once it's merged so the status doesn't get checked.